### PR TITLE
main: --roles-<LANG>.<KIND>, an option for enabling disabling roles

### PR DESCRIPTION
**Pull request by masatake**
Introduces `--roles-<LANG>.<KIND>=[+|-]roles` option.

There was no way to disable roles though all roles are enabled when defining.
With growing the number of roles, we may need the way to turn off (and on) the rules.

### CODE REVIEW

**Comment by masatake** [comment]
TODO:

* align the form of `--_roledef-<LANG>=kind.role,desc` to that of `--roles-<LANG>.<KIND>`: `--_roledef-<LANG>.<KIND>=role,deck`.

* align the form of `--_param-<LANG>:param=value` to that of `--roles-<LANG>.<KIND>`: `--_param-<LANG>.param=value`.

* update optlibc2c to handle `--roles-<LANG>.<KIND>` option.

**Comment by masatake** [comment]
The man page must be got reviewing.

File: Tmain/errors-in-options-roles.d/run.sh [hunk 1]
```diff
@@ -0,0 +1,121 @@
+# Copyright: 2020 Masatake YAMATO
+# License: GPL-2
+CTAGS=$1
+
+CTAGS=$1
```
Comment (k-takata) [inline Tmain/errors-in-options-roles.d/run.sh:5]: Same as above line. Mistake?

Comment (masatake) [reply Tmain/errors-in-options-roles.d/run.sh:5]: Mistake. Thank you.

Comment (masatake) [reply Tmain/errors-in-options-roles.d/run.sh]: Fixed. thank you.

**Pull request merged**